web/modules/custom/dpl_breadcrumb/dpl_breadcrumb.module

```diff
@@ -186,7 +186,39 @@ function dpl_breadcrumb_node_presave(Node $node): void {
   $breadcrumb_parent = $service->getStructureParent($breadcrumb_item);
 
   $node->set($field_name, [$breadcrumb_parent?->id()]);
+}
+
+/**
+ * Make sure nodes are re-saved when placed in the breadcrumb tree.
+ *
+ * On some node types, we have a structure field, that allows the editor to
+ * place the node as a child of an term.
+ * However, when the actual node is placed in the tree as a parent (as in, a
+ * term has been created, that links to this node), we want to take this
+ * editor control away, and automatically update the field.
+ * This happens in dpl_breadcrumb_node_presave().
+ * However, if you first setup the node with all it's content, and then go
+ * add it to the tree, no save is triggered on the node, and then it ends up
+ * with a "broken" state.
+ * This hook's task, is to run save() on nodes that have been placed in
+ * the tree.
+ */
+function dpl_breadcrumb_taxonomy_term_presave(TermInterface $term): void {
+  $service = DrupalTyped::service(BreadcrumbHelper::class, 'dpl_breadcrumb.breadcrumb_helper');
 
+  if ($term->bundle() !== $service->getStructureVid() || !$term->hasField('field_content')) {
+    return;
+  }
+
+  $contents = $term->get('field_content')->referencedEntities();
+
+  foreach ($contents as $content) {
+    if (!($content instanceof FieldableEntityInterface)) {
+      continue;
+    }
```

Comment:
Consider if this check is necessary.

As I see it `referencedEntities()` will return `EntityInterface[]` and `EntityInterface` declares the `save()` method.